src/index.js

```diff
@@ -1,12 +1,15 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import "./index.css";
-import App from "./App";
+import App, { appRouter } from "./App";
+import { RouterProvider } from "react-router-dom";
 
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
   <React.StrictMode>
-    <App />
+    <RouterProvider router={appRouter}>
+      <App />
+    </RouterProvider>
   </React.StrictMode>
```

Comment:
why this is used?

Reply:
For our appRouter config to be applied to the app